### PR TITLE
Remove requiring credentials to be in a file/env var

### PR DESCRIPTION
**Pull request by jmesterh**
Let the library figure out the credentials on it's own. 
Setting GOOGLE_APPLICATION_CREDENTIALS still works, if you still want to force it to use a json token.

### CODE REVIEW

**Comment by wenchengl** [comment]
the earlier code was for future support for using multiple service accounts in one Java  process.  You can merge this commit and keep the old code as comment, I will rework on this in the future.

**Comment by jmesterh** [comment]
Good point. I added back in support for loading from a file, but now it's optional.

**Comment by garricko** [comment]
What is the purpose of all the Avro related changes in this PR?

File: src/main/java/com/github/susom/dbgoodies/etl/Etl.java [hunk 1]
```diff
@@ -272,36 +272,35 @@ SaveAsAvro fetchSize(int nbrRows) {
...
-        Optional<String> google_credential_file = Optional.ofNullable(System.getenv("GOOGLE_APPLICATION_CREDENTIALS")) ;
-        if(!google_credential_file.isPresent()){
-            log.error("Can not get google api credential file from environment variable GOOGLE_APPLICATION_CREDENTIALS ");
-            System.exit(1);
-        }
+      Map<String, String> labels = new HashMap<>();
+
+      Optional<String> google_credential_file = Optional.ofNullable(System.getenv("GOOGLE_APPLICATION_CREDENTIALS")) ;
+
+      labels.put("job", "testing");
```
Comment (garricko) [inline src/main/java/com/github/susom/dbgoodies/etl/Etl.java]: Probably shouldn't be hard-coding the labels

Comment (jmesterh) [reply src/main/java/com/github/susom/dbgoodies/etl/Etl.java]: I didn’t make any Avro changes... probably Idea auto-formatting the code. The spacing was all over the place.

Comment (jmesterh) [reply src/main/java/com/github/susom/dbgoodies/etl/Etl.java]: If you go into the diff settings above and check "ignore whitespace" you will see it's all whitespace... I try to catch that before I commit but hitting cmd-option-L is practically a nervous tick of mine. Do we have a coding standard somewhere? eg. 2 vs 4 spaces for indentation....

**Comment by garricko** [comment]
Coding standards for all of our Java projects follow the Google standards unless otherwise noted (https://google.github.io/styleguide/javaguide.html).

File: src/main/java/com/github/susom/dbgoodies/etl/Etl.java (hunk 1 above)
Comment (wenchengl) [reply src/main/java/com/github/susom/dbgoodies/etl/Etl.java]: removed line 286 and committed to this pr

**Pull request merged**